sycl/include/CL/__spirv/spirv_ops.hpp

```diff
@@ -311,6 +311,27 @@ extern __DPCPP_SYCL_EXTERNAL void __spirv_CooperativeMatrixPrefetchINTEL(
     T *Ptr, uint32_t NumRows, uint32_t NumCols, unsigned int CacheLevel,
     __spv::MatrixLayout Layout, size_t Stride);
 
+template <typename T, typename Tp, std::size_t R, std::size_t C,
+          __spv::MatrixUse U,
+          __spv::MatrixLayout L = __spv::MatrixLayout::RowMajor,
+          __spv::Scope::Flag S = __spv::Scope::Flag::Subgroup>
+extern __DPCPP_SYCL_EXTERNAL
+    __spv::__spirv_JointMatrixINTEL<Tp, R, C, L, S, U> *
+    __spirv_CooperativeMatrixLoadOffsetINTEL(T *Ptr, int32_t RowIndex,
+                                             int32_t ColIndex,
+                                             __spv::MatrixLayout Layout = L,
+                                             std::size_t Stride = 0,
+                                             int MemOperand = 0);
+
+template <typename T, typename Tp, std::size_t R, std::size_t C,
+          __spv::MatrixUse U,
+          __spv::MatrixLayout L = __spv::MatrixLayout::RowMajor,
+          __spv::Scope::Flag S = __spv::Scope::Flag::Subgroup>
+extern __DPCPP_SYCL_EXTERNAL void __spirv_CooperativeMatrixStoreOffsetINTEL(
+    T *Ptr, int32_t RowIndex, int32_t ColIndex,
+    __spv::__spirv_JointMatrixINTEL<Tp, R, C, L, S, U> *Object,
```

Comment:
This line would also need to be updated to `__spv::__spirv_CooperativeMatrixKHR<Tp, S, R, C, U> *Object` for the same reason above.